examples/30_extended/study_tutorial.py

```diff
@@ -68,7 +71,7 @@
 )
 print(evaluations.head())
 
-############################################################################
+###########################################################from openml.testing import cat, cont#################
```

Comment:
The `from openml.testing import cat, cont` looks misplaced?

Reply:
Fixed.